### DISCUSSION TITLE
关于 VMess MD5 认证信息淘汰机制的疑问

### DISCUSSION THREAD

**Discussion by wangshengxin1998** [v2fly/v2ray-core#1308] [Q&A] [answered]
从2022年开始,服务端就不再接受旧版vmess请求,那么客户端到时候会不会强制以vmessAEAD连接到服务器?

**Comment by AkinoKaede** [comment, marked as answer]
不会，当 alertID >0 时，VMess 出站仍将使用 VMess MD5。

**Reply by wangshengxin1998** [reply]
希望到时候最新版本作为客户端能强制以vmessAEAD连接到服务器,来实现平滑过渡
